posthog-core/src/index.ts

```diff
@@ -1225,7 +1249,7 @@ export abstract class PostHogCore extends PostHogCoreStateless {
     return this._decideResponsePromise
   }
 
-  private setKnownFeatureFlags(featureFlags: PostHogDecideResponse['featureFlags']): void {
+  private setKnownFeatureFlags(featureFlags: PostHogDecideResponse['featureFlags'] | null): void {
```

Comment:
since this and `setKnownFeatureFlagPayloads` accept `null` now, we can change the other places from:
```
this.setKnownFeatureFlags({})
this.setKnownFeatureFlagPayloads({})
```

to null values as well.

Reply:
I did this I believe; I grepped those methods and I am using `null` in the only place they're called.